css/modules/icons.scss

```diff
@@ -1,28 +1,59 @@
+                }
+            }
+        }
+    }
+
+    > nav {
+        a {
+            text-align: center;
+            background-size: 24px 24px;
+            background-repeat: no-repeat;
+            background-position: top center;
+            padding: 0 0 0 52px;
+            font-size: $font-size-small;
+            line-height: 1rem;
+        }
+
+        .about    {
+            background-image: url('images/alpha_icons/biscay/searchlocation.svg');
+        }
+
+        .news     {
+            background-image: url('images/alpha_icons/biscay/news.svg'          );
+        }
+
+        .city     {
+            background-image: url('images/alpha_icons/biscay/factory.svg'       );
+        }
+
+        .services {
+            background-image: url('images/alpha_icons/biscay/services.svg'      );
+        }
     }
-    .about    { background-image:url('images/alpha_icons/biscay/searchlocation.svg'); }
-    .news     { background-image:url('images/alpha_icons/biscay/news.svg'          ); }
-    .city     { background-image:url('images/alpha_icons/biscay/factory.svg'       ); }
-    .services { background-image:url('images/alpha_icons/biscay/services.svg'      ); }
 }
 @media all and (min-width:$viewport-medium) {
-    #nav1 {
+    header > nav {
```

Comment:
Here is another place where it might make sense to continue to use the ID selector.  We control this ID in our custom theme; plus, there may come a time where there's more than one nav element in our header.